docs/tutorial/tutorial.md

```diff
@@ -98,6 +100,8 @@ return React.createElement('div', {className: 'shopping-list'},
 );
 ```
 
+https://facebook.github.io/react/docs/react-api.html#createelement
```

Comment:
OK, I will close this pull request, but I will open another, better request. Thanks for activity, @gaearon I'm big your fan (I'm 11 years old and now I learning to develop with react)...